exporters/scss/src/js/generators/simple.ts

```diff
@@ -116,9 +116,9 @@ export function generateSimple(
         .map((stop) => `${formatColor(stop.color)} ${(Math.round(stop.position * 10) / 10) * 100}%`)
         .join(', ')})`;
     } else if (token.tokenType === 'Border') {
-      const styleProp = token.properties.find((prop) => prop.codeName === 'style');
-      if (styleProp && styleProp.value.length > 0) {
-        value = styleProp.value;
+      const styleProp = token.propertyValues.style;
+      if (styleProp) {
+        value = styleProp;
       } else {
         value = printUnit(token.value.width.measure, token.value.width.unit);
```

Comment:
`value = styleProp ??  printUnit(token.value.width.measure, token.value.width.unit);` ? But it's about preferences... 🤷🏻